lib/buffered_logger.rb

```diff
@@ -45,7 +41,17 @@ def started?
   end
 
   def sweep
-    @logdev.sweep
+    warn "[DEPRECATION] `sweep` is deprecated.  It is no longer necessary."
```

Comment:
I think that if we're bumping this a major version here, we don't really need to deprecate, we can just delete these.  Thoughts?